ui/packages/consul-peerings/app/components/consul/peer/form/generate/index.hbs

```diff
@@ -47,6 +47,19 @@
       </fsm.State>
 
       <fsm.State @matches={{'success'}}>
+        {{yield (hash
+          Fieldsets=(component "consul/peer/form/token/fieldsets"
+            item=@item
+            token=fsm.state.context.PeeringToken
+            regenerate=@regenerate
+            onclick=(queue (set @item 'Name' '') (fn fsm.dispatch "RESET"))
```

Comment:
I think we got rid of the RESET for the generate form, correct? Do we still need it though? Why do we dispatch `RESET` here? Will we exit the success state without handling `RESET` when we click?

Reply:
Oh good spot, yeah I think we can get rid of this seeing as we got rid of it in another PR, sorry! Gimme 2 tics